halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/google/GoogleAddAccountCommand.java

```diff
@@ -65,6 +65,13 @@ protected String getProviderName() {
   )
   private String userDataFile;
 
+  @Parameter(
+      names = "--regions",
+      variableArity = true,
+      description = "A list of regions for caching and mutating calls. This overwrites any default-regions set on the provider."
```

Comment:
There should be some GoogleProperties class that you can put this constant in and reference, it seems to appear in a few places in this PR.